consensus/polybft/bridge_event_manager.go

```diff
@@ -542,24 +563,47 @@ func (b *bridgeEventManager) multicast(msg interface{}) {
 func (b *bridgeEventManager) GetLogFilters() map[types.Address][]types.Hash {
 	var bridgeMessageResult contractsapi.BridgeMessageResultEvent
 
+	var bridgeMsg contractsapi.BridgeMsgEvent
+
 	return map[types.Address][]types.Hash{
-		b.config.bridgeCfg.ExternalGatewayAddr: {types.Hash(bridgeMessageResult.Sig())},
+		b.config.bridgeCfg.ExternalGatewayAddr: {
```

Comment:
This function is the part of the `EventSubscriber` on the `EventProvider` which we use for internal (blade) originated events, so we use this for internal contracts:

```suggestion
		b.config.bridgeCfg.InternalGatewayAddr: {
```